web-ui/src/main/resources/catalog/views/default/less/gn_map_default.less

```diff
@@ -97,7 +97,7 @@
       margin: 0;
       padding: 0;
       position: absolute;
-      width: 16.5em;
+      width: 24.5em; // 34em - 7em - 2.5em
```

Comment:
@MichelGabriel are the commented values useful or can be removed?

Reply:
They are a bit useful, maybe I should have added:
`maptools panel width` - `label width` - `panel tools opener button width`

Reply:
It's an indication where the width is coming from

Reply:
```suggestion
      width: 24.5em; // value = maptools panel width - label width - panel tools opener button width (34em - 7em - 2.5em)
```